plugins/SegmentEditor/angularjs/segment-generator/segmentgenerator.directive.less

```diff
@@ -1,5 +1,9 @@
 .segment-generator {
-  width: 900px;
+  width: 930px;
```

Comment:
Just out of curiosity: Did you extend the width, so the options in the top of the selector are displayed on one line only? If so keep in mind, that other languages might hold even longer translations, that might wrap into multiple lines again. In German it currently looks like this:
![dev matomo io_index php_module=CoreHome action=index idSite=1 period=day date=yesterday](https://user-images.githubusercontent.com/1579355/134150609-1d4a482e-14f3-49a9-b948-cc099846eb5f.png)